web/themes/custom/novel/novel.theme

```diff
@@ -35,6 +38,25 @@ function novel_theme(
   ];
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * @param mixed[] $variables
+ *   The variables for the theme hook.
+ */
+function novel_preprocess_dpl_react_app__search_header(array &$variables): void {
+  $search_svg = file_get_contents(sprintf('%s/%s',
+    \Drupal::theme()->getActiveTheme()->getPath(),
+    '/assets/dpl-design-system/icons/collection/Search.svg'
+  ));
```

Comment:
If we end of doing more of this stuff it would be nice with Twig functions to help inline a file.

Reply:
yes